src/pages/studyView/table/GeneTable.tsx

```diff
@@ -399,88 +423,49 @@ export class GeneTable extends React.Component<GeneTableProps, {}> {
         return !!this.props.cancerGeneFilterEnabled && this.props.filterByCancerGenes;
     }
 
+    @computed get allSelectedRowsKeys() {
+        return [...this.selectedRowsKeys, ...this.preSelectedRowsKeys];
```

Comment:
is there any chance these overlap? i.e., its okay not to use _.union?

Reply:
there is no chance. even if there is overlap it should be a problem, its just for lookup